Maple2.Model/Game/InteractObject.cs

```diff
@@ -91,15 +97,21 @@ public sealed class InteractGuildPosterObject : InteractObject<Ms2InteractDispla
 public sealed class InteractBillBoardObject : InteractObject<Ms2InteractMesh> {
     public override InteractType Type => InteractType.BillBoard;
 
-    public long OwnerId { get; init; }           // CharacterId
-    public string OwnerName { get; init; } = ""; // CharacterName
+    public readonly Character Owner;
```

Comment:
Probably better not to store Character here unless you need all the fields. The other values can easily get out of sync.